doc/source/data/monitoring-your-workload.rst

```diff
@@ -0,0 +1,80 @@
+.. _monitoring-your-workload:
+
+Monitoring Your Workload
+========================
+
+This section helps you debug and monitor the execution of your Ray Data :class:`~ray.data.Dataset` by viewing the:
+
+* Ray dashboard job detail page
+* Ray dashboard metrics
+* Ray Data logs
+
+Ray Data Dashboard
+------------------
+
+Ray Data emits Prometheus metrics in real-time while a Dataset is executing. These metrics are tagged by both dataset and operator, and are displayed in multiple views across the Ray dashboard.
+
+.. note::
+   Most metrics are only available for physical operators that use the map operation.
+
+Ray dashboard job detail page
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+For an overview of all datasets that have been running on your cluster, see the Ray Data Overview in the :ref:`job detail page <dash-jobs-view>`. This table appears once the first dataset starts executing on the cluster, and shows dataset details such as:
+
+* execution progress (measured in blocks)
+* execution state (running, failed, or finished)
+* dataset runtime
```

Comment:
What does runtime mean in this context?

Reply:
we show start/end time for a dataset, clarified this in latest commit